nifi-mock/src/main/java/org/apache/nifi/util/MockConfigurationContext.java

```diff
@@ -58,8 +58,12 @@ public MockConfigurationContext(final ControllerService service,
 
     @Override
     public PropertyValue getProperty(final PropertyDescriptor property) {
-        String value = properties.get(property);
```

Comment:
@Wesley-Lawrence I don't think this is really what we want here. The concept of a default value is 'use this if the value is null'. So changing this would be inconsistent with how the StandardProcessContext works as well. I think the more desirable approach would be to just update the SingleCharacterValidator being used such that we allow 0 or 1 character. Then, this would allow an empty string to be used as the delimiter, which I believe is equivalent.